### ISSUE TITLE
Custom title bar should be focusable via command

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#149739]
The custom title bar should be focusable via a command "Focus Title Bar", similar to the "Focus Status Bar" command, it should pass focus to the title bar and left / right navigation would do the focus movement.

I understand that all the component in the title bar are accessible via other routes, but this would be nice to smoothen out the accessibility experience.

**Comment by sbatten** [comment]
Verification Steps:
1) enable the custom title bar (use Web if on mac os to bring the menu bar into the mix)
2) run the focus title bar command (either via cmd palette or custom keybinding)
3) Try with menubar, command center, and layout controls enabled/disabled

The first entry visible item in the title bar should get focused
